src/aiida/__init__.py

```diff
@@ -94,7 +106,10 @@ def get_file_header(comment_char: str = '# ') -> str:
 
 
 def load_ipython_extension(ipython):
-    """Load the AiiDA IPython extension, using ``%load_ext aiida``."""
-    from .tools.ipython.ipython_magics import AiiDALoaderMagics
+    """Load the AiiDA IPython extension, using ``%load_ext aiida``.
+
+    :param ipython: InteractiveShell instance. If omitted, the global InteractiveShell is used.
```

Comment:
Thanks!

In this function, the `ipython` argument is not optional (it doesn't have a default value). So I think the docstring needs to be reworded

```suggestion
    :param ipython: InteractiveShell instance. If None, the global InteractiveShell is used.
```

Reply:
Thanks right, thanks for careful reviewing.